tasks.c

```diff
@@ -467,6 +467,9 @@ static void prvInitialiseTaskLists( void ) PRIVILEGED_FUNCTION;
  *
  */
 static portTASK_FUNCTION_PROTO( prvIdleTask, pvParameters ) PRIVILEGED_FUNCTION;
+#if ( configNUM_CORES > 1 )
+static portTASK_FUNCTION_PROTO( prvMinimalIdleTask, pvParameters ) PRIVILEGED_FUNCTION;
```

Comment:
There is no need to use portTASK_FUNCTION_PROTO - that is only useful on tiny parts or when a compiler has special requirements for functions that implement tasks - neither is going to be the case for SMP ports.  This can just be:
static void prvMinimalIdleTask( void *pvParameters ) PRIVILEGED_FUNCTION;

Reply:
Per our discussion, I left this use of the macro.  Some ports still need this macro to decorate the function.